controller/request_test.go

```diff
@@ -39,8 +38,6 @@ func Test_Request_NewRequest(t *testing.T) {
 
 	for i, testCase := range testCases {
 		newRequest := NewRequest(testCase.Input)
-		fmt.Printf("%#v\n", newRequest)
-		fmt.Printf("%#v\n", testCase.Expected)
```

Comment:
Just removed debug logging. Somebody forgot to remove before merging. Probably even myself.